nodejs/src/index.js

```diff
@@ -22,9 +26,18 @@ exports.write = function (struct, compression, callback) {
 };
 
 exports.read = function (Model, rawData, callback) {
+	var callback = arguments.length === 4 ? arguments[3] : arguments[2];
```

Comment:
Has this bit been written like this instead of:
`function (Model, rawData, disable, callback) {}`

To stop breaking changes?